### PR TITLE
WB/ES: Update release notes

### PR DESCRIPTION
**Pull request by mcimbora**

### CODE REVIEW

**Comment by mcimbora** [comment]
@ge0ffrey I'll update the REST urls once we settle down with a solution.

**Comment by mcimbora** [comment]
@ge0ffrey PR updated to reflect latest API changes from https://github.com/kiegroup/droolsjbpm-integration/pull/1098

File: download/releaseNotes/releaseNotes7.2.adoc [hunk 1]
```diff
@@ -0,0 +1,69 @@
+= Release Notes 7.2
+:awestruct-description: New and noteworthy, demos and status for OptaPlanner 7.2.
+:awestruct-layout: releaseNotesBase
+:awestruct-priority: 1.0
+:awestruct-release_notes_version: 7.2
+:awestruct-release_notes_version_qualifier: Final
```
Comment (ge0ffrey) [inline download/releaseNotes/releaseNotes7.2.adoc]: this should probably be Beta

File: download/releaseNotes/releaseNotes7.2.adoc [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+[[NewAndNoteWorthyEngine]]
+== New and noteworthy: Engine
+
+=== Construction Heuristic power tweaking made simpler
```
Comment (ge0ffrey) [inline download/releaseNotes/releaseNotes7.2.adoc]: Is this a copy-paste-forgot-to-delete from the 7.1 recipe?

File: download/releaseNotes/releaseNotes7.2.adoc [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  void addProblemFactChanges(String containerId,
+                             String solverId,
+                             List<ProblemFactChange> problemFactChange);
+
+  Boolean isEveryProblemFactChangeProcessed(String containerId,
+                                            String solverId);
+----
+
+==== REST API
+- POST /containers/containerId/solvers/solverId/problemfactchanges
```
Comment (ge0ffrey) [inline download/releaseNotes/releaseNotes7.2.adoc]: Can we add something like "new REST API" or "new methods" or "these endpoints have been added"? Now it's just a list, so it's not clear what the change is from 7.1 to 7.2.

Also, let's put the uri's between `backticks`?

File: download/releaseNotes/releaseNotes7.2.adoc [hunk 4]
```diff
@@ -0,0 +1,69 @@
+= Release Notes 7.2
+:awestruct-description: New and noteworthy, demos and status for OptaPlanner 7.2.
+:awestruct-layout: releaseNotesBase
+:awestruct-priority: 1.0
+:awestruct-release_notes_version: 7.2
```
Comment (ge0ffrey) [inline download/releaseNotes/releaseNotes7.2.adoc]: IIRC, the generated release notes page will have a link to upgrade recipe 7.2 page, which will give a 404 if it doesn't exist yet. Please create an empty one? Probably easiest to copy the 6.5 upgrade recipe - that's empty too and has some wording around it.

File: download/releaseNotes/releaseNotes7.2.adoc [hunk 5]
```diff
@@ -0,0 +1,69 @@
+= Release Notes 7.2
```
Comment (ge0ffrey) [inline download/releaseNotes/releaseNotes7.2.adoc]: Don't forget to adjust download/releaseNotes/index.adoc and download/upgradeRecipe/index.adoc

**Comment by mcimbora** [comment]
@ge0ffrey Thanks for the review Geoffrey, please update the engine N&N when you have some time.

**Pull request merged**